fuse_variables/include/fuse_variables/orientation_3d_stamped.h

```diff
@@ -91,15 +93,49 @@ class Orientation3DStamped final : public FixedSizeVariable<4>, public Stamped
    */
   explicit Orientation3DStamped(const ros::Time& stamp, const fuse_core::UUID& device_id = fuse_core::uuid::NIL);
 
+  template <typename T>
```

Comment:
Should these be moved to util? Doesn't really matter; just curious where you think the better home would be.

Reply:
Sure. It seems reasonable that someone somewhere might want this functionality outside of this variable.

Reply:
We'd have to make a `util.h` inside `fuse_variables`, though, because I use these methods inside a variable type, and if we depend on `fuse_constraints`, where the current `util.h` lives, it'll be a circular dependency.